sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -167,14 +175,15 @@ class MetricsReader(object):
   A :class:`MetricsReader` retrieves metrics from pipeline result,
   prepares it for publishers and setup publishers.
   """
-  publishers = []  # type: List[ConsoleMetricsPublisher]
+  publishers = []  # type: List[Any]
 
   def __init__(
       self,
       project_name=None,
       bq_table=None,
       bq_dataset=None,
       publish_to_bq=False,
```

Comment:
Do you think it would be good to have consistent parameter naming for influx and bq? Or we plan to abandon bq in future?

Reply:
It's likely, but there's no decision yet. What influx parameters do you think can be improved?

Reply:
Well, I can see that inconsitent is `publish_to_bq` vs `influxdb_options`, but if we plan to have only `influxdb_options` then it's ok. Possibility I see here is to have something more generic like `metrics_options` or `metrics_storage_options`. Then if changes like this will happen in the future, there will be less to change.
The other possibility is changing it in a way it will be changed in Java SDK as well.

Reply:
Let's postpone any changes until we decide what to do about BQ. If we decide to keep BQ, we'll think about improving parameter naming so that InfluxDB and BQ parameters could coexist well enough (the same applies to Java SDK). WDYT?

Reply:
Sounds good 👍